eng/Versions.props

```diff
@@ -11,7 +11,7 @@
   <PropertyGroup>
     <MicrosoftNETCoreILAsmVersion>6.0.0-preview.6.21352.12</MicrosoftNETCoreILAsmVersion>
     <MicrosoftNETCoreILDAsmVersion>6.0.0-preview.6.21352.12</MicrosoftNETCoreILDAsmVersion>
-    <MicrosoftDotNetGenAPIPackageVersion>7.0.0-beta.22166.1</MicrosoftDotNetGenAPIPackageVersion>
+    <MicrosoftDotNetGenAPIPackageVersion>7.0.0-beta.22514.3</MicrosoftDotNetGenAPIPackageVersion>
```

Comment:
Should there be a versions.detail.xml entry for this?  If so I am wondering if this would get updated with the arcade dependency flow and then this would have never been an issue.